doc/tools/coqrst/repl/coqtop.py

```diff
@@ -30,47 +30,49 @@ def __init__(self, err, last_sentence, before):
         self.last_sentence = last_sentence
 
 class CoqTop:
-    """Create an instance of coqtop.
+    """Create an instance of rocq top.
 
-    Use this as a context manager: no instance of coqtop is created until
-    you call `__enter__`.  coqtop is terminated when you `__exit__` the
+    Use this as a context manager: no instance of rocq top is created until
+    you call `__enter__`.  rocq top is terminated when you `__exit__` the
     context manager.
 
     Sentence parsing is very basic for now (a "." in a quoted string will
     confuse it).
 
     When environment variable COQ_DEBUG_REFMAN is set, all the input
-    we send to coqtop is copied to a temporary file "/tmp/coqdomainXXXX.v".
+    we send to rocq top is copied to a temporary file "/tmp/coqdomainXXXX.v".
     """
 
     COQTOP_PROMPT = re.compile("\r\n[^< \r\n]+ < ")
 
-    def __init__(self, coqtop_bin=None, color=False, args=None) -> str:
-        """Configure a coqtop instance (but don't start it yet).
+    def __init__(self, rocqbin=None, color=False, args=None) -> str:
+        """Configure a rocq top instance (but don't start it yet).
 
-        :param coqtop_bin: The path to coqtop; uses $COQBIN by default, falling back to "coqtop"
-        :param color:      When True, tell coqtop to produce ANSI color codes (see
-                           the ansicolors module)
-        :param args:       Additional arguments to coqtop.
+        :param rocqbin:    The path to rocq; uses $COQBIN by default, falling
+                           back to "rocq"
+        :param color:      When True, tell "rocq top" to produce ANSI color
+                           codes (see the ansicolors module)
+        :param args:       Additional arguments to "rocq top".
         """
-        if coqtop_bin and pexpect.utils.which(coqtop_bin):
-            self.coqtop_bin = coqtop_bin
-            self.args = (args or []) + ["-q"] + ["-color", "on"] * color
+        if rocqbin:
```

Comment:
do the same `rocqbin = rocqbin or ...` factorization as my comment on coqdoc

Reply:
done also, thank you